changelog/unreleased/issue-119

```diff
@@ -0,0 +1,6 @@
+Security: Fix Docker configuration for DISABLE_AUTHENTICATION
+
+A regression was introduced which caused the DISABLE_AUTHENTICATION environment variable to stop working in Docker configurations, this was due to a new flag being introduced to disable authentication --no-auth. 
```

Comment:
Please wrap lines after approx. 80 characters and split this sentence into two shorter ones.